src/moscot/problems/space/_mixins.py

```diff
@@ -282,30 +276,28 @@ def _subset_spatial(
     ) -> ArrayLike:
         if spatial_key is None:
             spatial_key = self.spatial_key
-        return self.adata[self.adata.obs[self._policy._subset_key] == k].obsm[spatial_key].astype(np.float_, copy=True)
+        return self.adata[self.adata.obs[self._policy._subset_key] == k].obsm[spatial_key].astype(float, copy=True)
 
     @staticmethod
     def _affine(
-        tmap: LinearOperator, tgt: ArrayLike, src: ArrayLike, forward: bool = True, *args: Any
+        tmap: LinearOperator,
+        src: ArrayLike,
+        tgt: ArrayLike,
     ) -> Tuple[ArrayLike, ArrayLike]:
         """Affine transformation."""
         tgt -= tgt.mean(0)
-        if forward:
-            out = tmap.dot(src)
-        else:
-            out = tmap.T.dot(src)
+        out = tmap @ src
         H = tgt.T.dot(out)
         U, _, Vt = svd(H)
         R = Vt.T.dot(U.T)
         tgt = R.dot(tgt.T).T
         return tgt, R
 
     @staticmethod
-    def _warp(tmap: LinearOperator, _: ArrayLike, src: ArrayLike, forward: bool = True) -> Tuple[ArrayLike, None]:
+    def _warp(tmap: LinearOperator, src: ArrayLike, tgt: ArrayLike) -> Tuple[ArrayLike, None]:
         """Warp transformation."""
-        if forward:
-            return tmap.dot(src), None
-        return tmap.T.dot(src), None
+        del tgt
```

Comment:
do we need tgt here and in the signature?

Reply:
Yes, if I wanna still pass it as `kwargs`, if not, then no.